README.md

```diff
@@ -1,103 +1,74 @@
-# listenbrainz-server
+listenbrainz-server
+===================
 
-Server for the ListenBrainz project.
+Server for the ListenBrainz project
 
-[Website](https://listenbrainz.org) | [Beta website](https://beta.listenbrainz.org) | [Bug tracker](https://tickets.metabrainz.org/projects/LB/issues)
+[Website](https://listenbrainz.org) |
+[Documentation](https://listenbrainz.readthedocs.io) |
+[Beta website](https://beta.listenbrainz.org) |
```

Comment:
I think we should remove this link considering it's been down for a long time now.